docs/guide/mocking.md

```diff
@@ -392,9 +392,9 @@ export function readHelloWorld(path) {
 ```
 
 ```ts
+import { fs, vol } from 'memfs'
```

Comment:
let's keep this rule out of markdown. The `vitest` import is intentionally the first one